tests/smoke-tests/defaults.test.ts

```diff
@@ -36,8 +36,9 @@ for (let packageManager of SUPPORTED_PACKAGE_MANAGERS) {
       addonDir = join(tmpDir, addonName);
       await execa({
         cwd: tmpDir,
+        extendEnv: false,
```

Comment:
extendEnv is noise in this PR without an accompanying `env: {}` to set the env to empty